distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/AbstractIntegrationTest.java

```diff
@@ -81,38 +82,113 @@
 
     protected static final String KARAF_VERSION = "2.4.3";
 
-    // TODO: Use the Camel AvailablePortFinder.getNextAvailable() test method
-    protected static final String HTTP_PORT = "9081";
+    protected static final String OPENSEARCH_SOURCE_ID = "openSearchSource";
 
-    protected static final String HTTPS_PORT = "9993";
+    protected static final String CSW_SOURCE_ID = "cswSource";
 
-    protected static final String SSH_PORT = "9101";
+    protected static ServerSocket placeHolderSocket;
 
-    protected static final String RMI_SERVER_PORT = "44445";
+    protected static Integer basePort;
 
-    protected static final String RMI_REG_PORT = "1100";
+    protected static final String DDF_HOME_PROPERTY = "ddf.home";
 
-    protected static final String SECURE_ROOT = "https://localhost:";
+    protected static String ddfHome;
 
-    protected static final String SERVICE_ROOT = SECURE_ROOT + HTTPS_PORT + "/services";
+    public enum DynamicPort {
+        BASE_PORT("org.codice.ddf.system.basePort"), HTTP_PORT(
```

Comment:
This is a system property just used for tests. The baseport is used to define an offset that the test can then use to assign its ports without interfering with other tests.